### DISCUSSION TITLE
If installing using JBang is there any way to get the fixes without having to wait for a new release?

### DISCUSSION THREAD

**Discussion by Bugmaniac71** [ebullient/ttrpg-convert-cli#37] [General]
If installing using JBang is there any way to get the fixes without having to wait for a new release?

**Comment by ebullient** [comment]
yes.. I clear the jitpack snapshot after every push, which would allow it to be rebuilt.

This should do it: 

```shell
jbang app install --name 5e-convert-snap --force --fresh https://jitpack.io/dev/ebullient/json5e-convert-cli/199-SNAPSHOT/json5e-convert-cli-199-SNAPSHOT-runner.jar
```

Note there may be a pause if you're the first to request the snapshot, as it is built on demand.

**Comment by Bugmaniac71** [comment]
Thanks Erin I'll try that later.

**Comment by ebullient** [comment]
I've added this to the README as well. ;)

**Comment by Bugmaniac71** [comment]
Working perfectly now thanks!
